encord/client.py

```diff
@@ -296,6 +296,23 @@ def add_private_data_to_dataset(
 
         return AddPrivateDataResponse.from_dict(response)
 
+    def update_data_item(self, dataset_hash: str, data_item: Video):
```

Comment:
Might be a bit misleading with the `Video`. We have to explain that we cannot change the data_hash and the file_link. Maybe we should just introduce a helper dataclass such as `UpdateableDataFields` (or another more legible name). What do you think?

Reply:
That sounds reasonable. Good catch.